lib/yelp/responses/models/business.rb

```diff
@@ -0,0 +1,26 @@
+require 'yelp/responses/base'
+require 'yelp/responses/models/deal'
+require 'yelp/responses/models/gift_certificate'
+require 'yelp/responses/models/location'
+require 'yelp/responses/models/review'
+
+module Yelp
+  module Response
+    module Model
+      class Business < Response::Base
+        attr_accessor :categories, :deals, :display_phone, :distance, :eat24_url, :gift_certificates, :id, :image_url,
```

Comment:
Thought about it more and I think `attr_reader` makes more sense here. People shouldn't be changing the response values and may lead to confusion. I changed everything to use `attr_reader`